meson.build

```diff
@@ -101,6 +101,12 @@ if ['linux', 'android', 'ios', 'darwin'].contains(system)
   elif cpu_family == 'riscv64'
     # We dont't have riscv64-specific optimization for now.
     asm_format = asm_format64
+  elif cpu_family == 'ppc'
+    # We dont't have powerpc-specific optimization for now.
```

Comment:
```suggestion
    # We don't have powerpc-specific optimization for now.
```

Reply:
Thanks, Ryan, I just copy-pasted a note about RISC-V without checking the spelling. It is worth fixing that line perhaps as well then.